lib/actions/google/drive/google_drive.js

```diff
@@ -44,6 +50,19 @@ class GoogleDriveAction extends Hub.OAuthAction {
             }
             const stateJson = JSON.parse(request.params.state_json);
             if (stateJson.tokens && stateJson.redirect) {
+                if (request.params.domain_allowlist) {
```

Comment:
a whitespace allowlist string would still trigger this, should we have a more robust condition before entering this?